examples/with-strict-csp/server.js

```diff
@@ -1,22 +0,0 @@
-const express = require('express')
-const next = require('next')
-const csp = require('./csp')
-
-const port = parseInt(process.env.PORT, 10) || 3000
-const dev = process.env.NODE_ENV !== 'production'
-const app = next({ dev })
-const handle = app.getRequestHandler()
-
-app.prepare()
-  .then(() => {
-    const server = express()
-
-    csp(server)
-
-    server.use(handle)
-
-    server.listen(port, (err) => {
-      if (err) throw err
-      console.log(`> Ready on http://localhost:${port}`)
-    })
-  })
```

Comment:
Are you planning to re-define an example with custom `_document` with strict CSP? I think that would be very helpful for people.

Reply:
Setting up CSP is now as simple as setting a header. I feel like an example for that would be a waste of time.